src/qt/askpassphrasedialog.cpp

```diff
@@ -238,10 +238,10 @@ bool AskPassphraseDialog::event(QEvent *event)
 void AskPassphraseDialog::toggleShowPassword(bool show)
 {
     ui->toggleShowPasswordButton->setDown(show);
-    const auto mode = show ? QLineEdit::Normal : QLineEdit::Password;
-    ui->passEdit1->setEchoMode(mode);
-    ui->passEdit2->setEchoMode(mode);
-    ui->passEdit3->setEchoMode(mode);
+    const auto echoMode = show ? QLineEdit::Normal : QLineEdit::Password;
+    ui->passEdit1->setEchoMode(echoMode);
+    ui->passEdit2->setEchoMode(echoMode);
+    ui->passEdit3->setEchoMode(echoMode);
```

Comment:
We don't do these changes, since it is non-trivial to prove the refactoring does the right thing:  #10136

Reply:
Wasn't the problem that people used ancient compilers?

This was one of the comments:

> And there is still value in compiling with Wshadow periodically (with a carefully chosen compiler) and seeing if it finds any bug.

I'm using a carefully chosen compiler :-)

Reply:
Oh, I misread a report. Sorry. Reverted the shadowing change :-)